bin/generate-billings.js

```diff
@@ -56,30 +56,56 @@ const InProgressBilling = {
 const SuccessfulBilling = {
     dv: DV,
     sender: SENDER,
-    name: 'Success',
-    shortDescription: 'Шаблон СБ Бизнес Онлайн 8_2',
-    detailsTitle: 'Подключение реестрового обмена',
-    detailsText: 'Выбрав данный вариант интеграции, вам будет необходимо загрузить ваши реестры к нам самостоятельно',
+    name: 'Интеграция через загрузку вашего реестра (DONE STATUS)',
+    shortDescription: 'Поддерживаемые форматы: 1С, СБ Бизнес Онлайн 8_2 и 9_1',
+    detailsTitle: 'Интеграция через загрузку реестров',
+    detailsText: 'Выберите формат, в котором хотите загружать ваши реестры в Домá.\n' +
+        'Мы запомним ваш выбор и в следующий раз не будем спрашивать про формат файлов.',
     detailsConfirmButtonText: 'Подключить',
-    detailsInstructionButtonText: 'Подробнее про шаблон',
-    detailsInstructionButtonLink: 'https://www.sberbank.ru/ru/s_m_business/new_sbbol',
     contextDefaultStatus: BILLING_INTEGRATION_ORGANIZATION_CONTEXT_FINISHED_STATUS,
     billingPageTitle: 'Биллинг, реестровый обмен',
-    dataFormat: Lvl1DataFormat,
+    dataFormat: Lvl3DataFormat,
+    availableOptions: {
```

Comment:
What if we have a billing integration which would require more in-depth setup? Like an input field with special API key?

Reply:
If we use a reusable solution I'd saw it like this

A form that is encoded into JSON, which is rendered on stage of creating a BillingContext and supplied to BillingContext.meta

Pros:
- It would cover potentially all cases
- It would be reusable right from the start (_in Registry Exchange billing form input_)

Cons:
- It would require more time (Code + Docs)
- Support or non-tech teammates wont be able to create billing context manually without extensive documentation

> Maybe use this https://nikolamargit.dev/dynamic-form-using-ant-design/

> I did not do any research but have a hypothesis that it's easy to find a library that does this form rendering

Reply:
I think support should be able to get / change it without lots of problems...Since there'll be some cases when clients will want to change billing formats

Reply:
Think like dynamic forms here are a bit yagni

Reply:
## Result after conversation:
We don't know for now, if we will even need something like this later.....
and we are not even sure that the current interface and flow will remain so

We decided to left this variant for MVP. And we'll change it later